microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -1,25 +1,21 @@
 ---
-title: Get started with WebView2 in WinUI 2 (UWP) apps (public preview)
+title: Get started with WebView2 in WinUI 2 (UWP) apps
```

Comment:
todo: remove prerelease throughout article, update captures

Reply:
For the WinUI 2 instructions: given that the WV2 NuGet is a dependency of the WinUI one, I am wondering if the developer can get away with just installing the WinUI 2 NuGet as this would save a step. Are there any cases where this wouldn't work?

Reply:
My latest commit moved "Install or update wv2 sdk" section to below "Install winui sdk" section.  
Next, I'm going to test the tutorial from scratch to check whether we can omit "install wv2 sdk" step/section.

Reply:
@adrastogi - testing now.  apparently installing WinUI but not WV2 then adding wv2 code gives: "error CS0234: The type or namespace name 'WebView2' does not exist in the namespace 'Microsoft.UI.Xaml.Controls' (are you missing an assembly reference?)".  Tent; testing...

Reply:
I'm going to leave this here partially just to remind myself -- @adrastogi's suggestion definitely is true for WinUI 3. A user actually hit errors because he installed both, leading to http://task.ms/40095882 and https://github.com/MicrosoftDocs/edge-developer/pull/2021 (which @mikehoffms merged so I know you know :-) ).

I would think it would also be true with WinUI 2 but I'll defer to testing with 2.8.

Reply:
In this PR, I updated the other article, about how to download the samples repo & update the Prerelease WinUI 2 SDK.  Those screenshots are updated to reflect latest Prerelease and the dependencies it installs:
https://review.docs.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2_sample_uwp?branch=pr-en-us-2043